### DISCUSSION TITLE
使用“发送到aria2”下载失败，提示“network error”

### DISCUSSION THREAD

**Discussion by dzhl** [AlistGo/alist#3738] [General]
选择目录或文件后，点击下载菜单中的“发送到aria2”会提示失败，截图如下，我已经在后头配置了aira地址，没有密匙
![image](https://user-images.githubusercontent.com/47837136/223133760-2892ad7e-b10f-4a74-99f0-772bbff21872.png)

**Comment by dzhl** [comment]
更新，在前台也需要设置aria2地址，已经可以正常发送地址到aria2中，但下载失败，提示如下，存储为最新版“”阿里云盘OPEN“”存储，
![image](https://user-images.githubusercontent.com/47837136/223135088-e2414984-c1e8-4321-be1a-d707b870c0f0.png)。另外使用rclone挂载alist出来的webdav，可以看到文件，但是复制也会出现类型问题，这是怎么回事呢？阿里云盘限制吗？

**Reply by sunnywangzi** [reply]
下载失败是因为aria2与下载服务器的tls握手失败，检查下载链接是否正常

**Comment by dzhl** [comment]
没有大佬知道怎么解决吗？

**Comment by longzusan** [comment]
遭遇同样问题

**Comment by nicktse1984** [comment]
遇到相同的問題，有大佬可以幫忙嗎?....

**Comment by sunnywangzi** [comment]
请确保aria2能被alist访问

**Reply by sunnywangzi** [reply]
前台和后台设置的aria2并不等同[官方文档有写](https://alist.nn.ci/zh/faq/why.html#两个aria2有什么不同)

**Comment by nicktse1984** [comment]
@sunnywangzi 

         非常感謝，終於解決了，需要在前臺設置好正確能訪問到Aria2的正確域名。
         
![image](https://user-images.githubusercontent.com/128733550/227410511-3c17939f-4889-47ab-81e4-0ee468bfff5a.png)

**Reply by xing03** [reply]
感谢，我的motrix默认的监听端口是16800
